locallibrary/catalog/tests/test_forms.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+from django.test import TestCase
+from django.utils import timezone
+from catalog.forms import RenewBookForm
+
+class RenewBookFormTest(TestCase):
+  def test_renew_form_date_field_label(self):
+    form = RenewBookForm()
+    self.assertTrue(form.fields['renewal_date'].label == None or
+    form.fields['renewal_date'].label == 'renewal date')
+  def test_renew_form_date_field_help_text(self):
```

Comment:
giữa 2 method cách ra 1 line trống em nhen
```suggestion
    form.fields['renewal_date'].label == 'renewal date')

  def test_renew_form_date_field_help_text(self):
```